.github/workflows/build_wheels.yml

```
@@ -24,7 +24,9 @@ jobs:
       - uses: actions/setup-python@v4
 
       - name: Install cibuildwheel
-        run: python -m pip install cibuildwheel==2.3.1
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install cibuildwheel==2.13.1
```

Is the pipe required?
```suggestion
        run:
          python -m pip install --upgrade pip
          python -m pip install cibuildwheel==2.13.1
```